server/routes/appointments/create/handlers/selectPrisoner.ts

```diff
@@ -1,12 +1,16 @@
 import { Request, Response } from 'express'
-import { Expose } from 'class-transformer'
+import { Expose, Type } from 'class-transformer'
 import { IsNotEmpty } from 'class-validator'
 import PrisonService from '../../../../services/prisonService'
+import { AppointmentType } from '../journey'
+import { Prisoner } from '../../../../@types/prisonerOffenderSearchImport/types'
+import { FormValidationError } from '../../../../formValidationErrorHandler'
 
 export class PrisonerSearch {
   @Expose()
-  @IsNotEmpty({ message: 'Enter a prisoner number to search by' })
-  number: string
+  @Type(() => String)
```

Comment:
I did this myself and was told not to include this @Type annotation as string is the default